editor.planx.uk/src/@planx/components/FileUploadAndLabel/schema.ts

```diff
@@ -65,18 +45,18 @@ export const ruleSchema: SchemaOf<Rule> = object({
 
 export const fileTypeSchema: SchemaOf<FileType> = object({
   name: string().required(),
-  fn: string().required(),
+  fn: string().required("Required field"),
```

Comment:
Without a more generic message, the error on deeply nested fields (for filetypes) becomes `filetype[3].fn is a required field`.